[2025-06-29T14:47:33.327+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2024-06-16T00:00:00+00:00 [queued]>
[2025-06-29T14:47:33.388+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2024-06-16T00:00:00+00:00 [queued]>
[2025-06-29T14:47:33.388+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-06-29T14:47:33.624+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2024-06-16 00:00:00+00:00
[2025-06-29T14:47:33.688+0000] {standard_task_runner.py:57} INFO - Started process 1882 to run task
[2025-06-29T14:47:33.714+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2024-06-16T00:00:00+00:00', '--job-id', '3209', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpmbl2j8oi']
[2025-06-29T14:47:33.748+0000] {standard_task_runner.py:85} INFO - Job 3209: Subtask delete_data_from_table
[2025-06-29T14:47:34.316+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2024-06-16T00:00:00+00:00 [running]> on host f561bfe7dd3a
[2025-06-29T14:47:34.843+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2024-06-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-16T00:00:00+00:00'
[2025-06-29T14:47:34.851+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2024-06-16' and dag_id = 'dag_with_postgres_operator_v03';
        
[2025-06-29T14:47:34.962+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T14:47:35.050+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T14:47:35.077+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2024-06-16' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2025-06-29T14:47:35.092+0000] {sql.py:427} INFO - Rows affected: 0
[2025-06-29T14:47:35.137+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20240616T000000, start_date=20250629T144733, end_date=20250629T144735
[2025-06-29T14:47:35.377+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-29T14:47:36.030+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
